### PR TITLE
update to ui-components 6

### PR DESCRIPTION
**Pull request by hobbitronics**
- update ui-components and sass
- install tslib

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@rollup/plugin-node-resolve": "^13.0.6",
     "@rollup/plugin-typescript": "^8.3.0",
     "@roxi/routify": "^2.18.3",
-    "@silintl/ui-components": "^5.0.1",
+    "@silintl/ui-components": "^6.0.0",
```
Comment (forevermatt) [inline package.json:27]: Does this change mean that we will now need to fix various aspects of our markup (e.g. `class="mdc-list flex"` to `class="mdc-deprecated-listflex"`)?

Comment (hobbitronics) [reply package.json:27]: I think you mean if we are styling using the mdc class names for styling or building new components. I am just confused by the lack of a space before flex. Is that a typo?

Comment (forevermatt) [reply package.json:27]: No, that was copied from [a change included in the ui-components 6.0.0 PR](https://github.com/silinternational/ui-components/pull/141/files#diff-c1cf8407ca53b75f6fd3c95697856f858b9bfd848733441720f2e3ab1af6e2bcR127), as an example of the CSS class name changes.

Comment (forevermatt) [reply package.json:27]: But yes, in response to the first part of your comment: This is if we are using the mdc class names for styling anywhere in cover-ui.

**Review by forevermatt — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (hobbitronics) [reply package.json:27]: > No, that was copied from [a change included in the ui-components 6.0.0 PR](https://github.com/silinternational/ui-components/pull/141/files#diff-c1cf8407ca53b75f6fd3c95697856f858b9bfd848733441720f2e3ab1af6e2bcR127), as an example of the CSS class name changes.

I think that was a find and replace error. I will want to fix that.

Comment (hobbitronics) [reply package.json:27]: > Does this change mean that we will now need to fix various aspects of our markup (e.g. `class="mdc-list flex"` to `class="mdc-deprecated-listflex"`)?

We are using the mdc-list class in _typography.scss so thanks for raising this question. I will need to update that.

**Review by forevermatt — APPROVED** [review]

**Pull request merged**